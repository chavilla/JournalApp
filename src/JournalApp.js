import React, { useState } from "react";
import { Provider } from "react-redux";
import { ShowContext } from "./components/context/showContext";
import AppRouter from "./components/routers/AppRouter";
import { store } from "./store/store";

const JournalApp = () => {
  const [show, setShow] = useState(false);

  return (
    <Provider store={store}>
      <ShowContext.Provider
        value={{
          show,
          setShow,
        }}
      >
        <AppRouter />
      </ShowContext.Provider>
    </Provider>
  );
};

export default JournalApp;
