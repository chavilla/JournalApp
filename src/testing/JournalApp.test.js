import React from 'react';
import {shallow} from 'enzyme';
import JournalApp from '../JournalApp'

describe('JournalApp component', () => {
    test('should to display the component', () => {
        const wrapper=shallow(<JournalApp />);
        expect(wrapper).toMatchSnapshot();
    })
    
})