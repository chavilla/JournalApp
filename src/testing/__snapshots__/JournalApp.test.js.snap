// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`JournalApp component should to display the component 1`] = `
<Provider
  store={
    Object {
      "dispatch": [Function],
      "getState": [Function],
      "replaceReducer": [Function],
      "subscribe": [Function],
      Symbol(observable): [Function],
    }
  }
>
  <ContextProvider
    value={
      Object {
        "setShow": [Function],
        "show": false,
      }
    }
  >
    <AppRouter />
  </ContextProvider>
</Provider>
`;
